.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/
 .env
 .phpunit.result.cache
+/libs/output-mapping/.env.local
```

Comment:
nechces to dat do `.gitignore` v `libs/output-mapping`? :)

Reply:
ah, to je nějakej renonc